fastlane/metadata/it/release_notes.txt

```diff
@@ -1 +0,0 @@
-- Corretto un bug che impediva lo spostamento della finestra dell'app
```

Comment:
Notice there's other three _deleted_ release notes locales.

That would be a problem in other apps because we wouldn't have localizations on ASC. Here, though, we have an issue that prevents store localization on the ASC end, so it doesn't matter whether or not we have up to date release notes 😞

Reply:
😒 😢